shell/etc/fish/conf.d/conda.fish

```diff
@@ -217,14 +217,17 @@ end
 #   command conda env list | awk 'NR > 2 {print $1}'
 # end
 
+set -x CONDA_ROOT_FISH_CACHED (command conda info --root)
```

Comment:
Use Python `sys.prefix` for faster results:

```
set -x CONDA_ROOT_FISH_CACHED (python -c "import sys; print(sys.prefix)")
```

Reply:
I stand corrected; some users (see here: https://github.com/conda/conda/issues/2611#issuecomment-340639349) might have another Python in `$PATH` and only use `conda`'s with `conda activate root`. So maybe that's not a good idea for everyone...

Reply:
Indeed. Besides `CONDA_ROOT_FISH_CACHED` is set only once at the start of a new shell and this is fast enough on all the machines where I tried (considering that most users are likely to have way more configuration code anyway).